tests/test_access_token.py

```diff
@@ -0,0 +1,102 @@
+import asyncio
+from typing import Callable, Type, Union
+
+import pytest
+import respx
+
+from huntflow_api_client import HuntflowAPI
+from huntflow_api_client.errors import InvalidAccessTokenError, TokenExpiredError
+from tests.fixtures.huntflow import Huntflow, TokenTypes
+
+
+@respx.mock
+async def test_valid_access_token__ok(
+    fake_huntflow: Huntflow,
+    huntflow_api_factory: Callable[[bool], HuntflowAPI],
+) -> None:
+    huntflow_api = huntflow_api_factory(False)
+
+    token = await huntflow_api._token_proxy._storage.get()  # type: ignore[attr-defined]
+    fake_huntflow.add_token(token.access_token)
+
+    response = await huntflow_api.request("GET", "/me")
+
+    assert response.status_code == 200
+    assert fake_huntflow.me_route.called
+
+
+@pytest.mark.parametrize(
+    ("unauthorized_token_type", "authorization_error"),
+    [
+        (TokenTypes.INVALID_TOKEN, InvalidAccessTokenError),
+        (TokenTypes.EXPIRED_TOKEN, TokenExpiredError),
+    ],
+)
+@respx.mock
+async def test_authorization_error__error(
```

Comment:
`test_authorization_error__error` why did you use the name with double underscore?
Seems like `test_authorization_error` is enough

Reply:
after the double underscore, I indicate the expected test result